src/platforms/javascript/guides/wasm/index.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Wasm
+sdk: sentry.javascript.wasm
+redirect_from:
+  - /platforms/javascript/wasm/
+  - /clients/wasm/
+  - /sdks/react/
+  - /platforms/wasm/
+description: "Learn how to use Sentry's Wasm integration and how it automatically reports errors and exceptions in your Wasm module."
+---
```

Comment:
Consider adding `tags:` or something else to include the word `WebAssembly`, so that this page will more likely show up in searches. Alternatively or additionally, we could add an entry like this one https://github.com/getsentry/sentry-global-search/commit/dfe4d231e9772fdc0c0855c7d137226b1a9a8abb to tell the search that Wasm and WebAssembly are synonyms.

Reply:
Added